controllers/main.py

```diff
@@ -16,31 +16,35 @@ class NeloController(http.Controller):
     _confirm_url = '/payment/nelo/confirm'
     _cancel_url = '/payment/nelo/cancel'
 
-    def _nelo_auth_payment(self, **post):
-        claims = self._get_claims(post['checkoutToken'])
+    def _nelo_auth_payment(self, **params):
+        claims = self._get_claims(params['checkoutToken'])
 
         if claims.get('order_id'):
             acquirer = request.env['payment.acquirer'].sudo().search([('provider', '=', 'nelo')])
             payload = json.dumps({
-                'checkoutToken': post['checkoutToken']
+                'checkoutToken': params['checkoutToken']
             })
             headers = {
                 'Authorization': 'Bearer %s' % (acquirer.nelo_merchant_secret),
-                'Content-Type': 'application/json'
+                'Content-Type': 'application/json',
+                'x-device-platform': 'web',
+                'x-app-version-code': '1'
             }
             try:
-                url = '%s/charge/auth' % (acquirer._get_nelo_urls()['rest_url'])
+                url = '%s/payments/auth' % (acquirer._get_nelo_urls()['rest_url'])
                 response = requests.request("POST", url, headers=headers, data=payload)
                 _logger.info('Nelo - url requested %s' % url)
                 _logger.info('Nelo - response: %s' % response)
                 response.raise_for_status()
-                url = '%s/charge/capture' % (acquirer._get_nelo_urls()['rest_url'])
-                response = requests.request("POST", url, headers=headers, data=payload)
+                url = '%s/payments/%s/capture' % (acquirer._get_nelo_urls()['rest_url'], claims.get('sub'))
```

Comment:
So, as discussed, let's use the uuid returned from `/payments` rather than getting it from the checkout token

Reply:
Done!